apps/web/test/utils/bookingScenario.ts

```diff
@@ -0,0 +1,742 @@
+  /**
+   * Prisma would return these eventTypes
+   */
+  eventTypes: InputEventType[];
+  /**
+   * Prisma would return these users
+   */
+  users: InputUser[];
+  /**
+   * Prisma would return these apps
+   */
+  apps?: App[];
+  bookings?: InputBooking[];
+  webhooks?: InputWebhook[];
+};
+
+type InputCredential = typeof TestData.credentials.google;
+
+type InputSelectedCalendar = typeof TestData.selectedCalendars.google;
+
+type InputUser = typeof TestData.users.example & { id: number } & {
+  credentials?: InputCredential[];
+  selectedCalendars?: InputSelectedCalendar[];
+  schedules: {
+    id: number;
+    name: string;
+    availability: {
+      userId: number | null;
+      eventTypeId: number | null;
+      days: number[];
+      startTime: Date;
+      endTime: Date;
+      date: string | null;
+    }[];
+    timeZone: string;
+  }[];
+};
+
+type InputEventType = {
+  id: number;
+  title?: string;
+  length?: number;
+  offsetStart?: number;
+  slotInterval?: number;
+  minimumBookingNotice?: number;
+  /**
+   * These user ids are `ScenarioData["users"]["id"]`
+   */
+  users?: { id: number }[];
+  hosts?: { id: number }[];
+  schedulingType?: SchedulingType;
+  beforeEventBuffer?: number;
+  afterEventBuffer?: number;
+  requiresConfirmation?: boolean;
+};
+
+type InputBooking = {
+  userId?: number;
+  eventTypeId: number;
+  startTime: string;
+  endTime: string;
+  title?: string;
+  status: BookingStatus;
+  attendees?: { email: string }[];
+};
+
+const Timezones = {
+  "+5:30": "Asia/Kolkata",
+  "+6:00": "Asia/Dhaka",
+};
+
+function addEventTypes(eventTypes: InputEventType[], usersStore: InputUser[]) {
+  const baseEventType = {
+    title: "Base EventType Title",
+    slug: "base-event-type-slug",
+    timeZone: null,
+    beforeEventBuffer: 0,
+    afterEventBuffer: 0,
+    schedulingType: null,
+
+    //TODO: What is the purpose of periodStartDate and periodEndDate? Test these?
+    periodStartDate: new Date("2022-01-21T09:03:48.000Z"),
+    periodEndDate: new Date("2022-01-21T09:03:48.000Z"),
+    periodCountCalendarDays: false,
+    periodDays: 30,
+    seatsPerTimeSlot: null,
+    metadata: {},
+    minimumBookingNotice: 0,
+    offsetStart: 0,
+  };
+  const foundEvents: Record<number, boolean> = {};
+  const eventTypesWithUsers = eventTypes.map((eventType) => {
+    if (!eventType.slotInterval && !eventType.length) {
+      throw new Error("eventTypes[number]: slotInterval or length must be defined");
+    }
+    if (foundEvents[eventType.id]) {
+      throw new Error(`eventTypes[number]: id ${eventType.id} is not unique`);
+    }
+    foundEvents[eventType.id] = true;
+    const users =
+      eventType.users?.map((userWithJustId) => {
+        return usersStore.find((user) => user.id === userWithJustId.id);
+      }) || [];
+    return {
+      ...baseEventType,
+      ...eventType,
+      workflows: [],
+      users,
+    };
+  });
+
+  logger.silly("TestData: Creating EventType", eventTypes);
```

Comment:
Yeah right now you need to set the log level manually in code like this 
`logger.setSettings({ minLevel: "silly" })`. Don't want to hardcode this, maybe I can pass some argument to vitest and then based on that I can enable this log level, so that it doesn't spam by default.

Reply:
We can maybe use an .env variable to set the logger level and keep the default to warn and error level.